spec/models/candidate_interface/degree_form_spec.rb

```diff
@@ -78,7 +78,6 @@
     it 'creates an array of objects based on the provided ApplicationForm' do
```

Comment:
We could dispose of the explicit ID at line 266 in a similar way — might need to fix up the lets a bit

Reply:
Done in the commit about refactoring the `DegreeForm` and spec.